concent_api/core/tests/test_core_views.py

```diff
@@ -355,6 +355,56 @@ def test_receive_return_http_204_if_no_messages_in_database(self):
         self.assertEqual(response.content.decode(), '')
         assert len(ReceiveStatus.objects.filter(delivered=False)) == 0
 
+    @freeze_time("2017-11-17 12:00:00")
+    def test_receive_should_get_ack_after_task_to_compute_is_not_after_deadline(self):
```

Comment:
Please give it a more sensible name: `test_receive_should_return_ack_if_the_receive_queue_contains_only_force_report_and_its_past_deadline`.

The current name makes no sense because `receive()` does not get anything. It only returns a message.